### PR TITLE
Add new module: Web Bruteforcer

### PR DESCRIPTION
**Pull request by Ernest1338**
Screenshot_20230906_005337
Screenshot_20230906_005240

### CODE REVIEW

**Review by GrzegorzSzczepanek — APPROVED** [review]
lgtm. After quick dissection I see no problems. Code seems clean and I assume you (our security specialist) were able to test it and eliminate bugs.

**Comment by Ernest1338** [comment]
I think I either added a TODO / commented on parts which I wasn't sure about.

File: lib/web_bruteforcer.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        if self.cli:
+            cli_print("Web content bruteforce finished", MessageType.INFO)
+
+        return self.results
+
+    # I *think* python will pass the wordlist by reference. This is quite important as this wordlist object
+    # can be multiple hundreds of Megabytes or even Gigabytes in size and cloning it for each thread is not
+    # ideal to say the least.
+
+    def scanning_thread(self, url: str, wordlist: list, starting_index: int, n_threads: int):
```
Comment (Ernest1338) [inline lib/web_bruteforcer.py:42]: I'm not sure if that `list` type is a correct way of expressint a list of strings. there is also `[str]` but Python doesn't seem to care about those type hints so when I get to setting up a type checker then we will know for sure

**Review by GrzegorzSzczepanek — CHANGES_REQUESTED** [review]
We're doing fine. So I'd add few more tasks on this todo list. My propositions for modules are:
-webscraper
-geolocate ip addresses
-DHCP listener
-Zip file passwords bruteforcer
-Cookie extractor
-proxy

we could do even more. BUt I think these are related to PixelToolkit idea and other projects

**Review by GrzegorzSzczepanek — APPROVED** [review]

**Comment by Ernest1338** [comment]
-webscraper :+1: 
-geolocate ip addresses :+1: (we could use a API for that)
-DHCP listener - how would that work?
-Zip file passwords bruteforcer - already in the TODO list
-Cookie extractor - i'm not sure how that would work as cookies are stored in a web browser and we would need to support every web browser and extract cookies from the browser storage files. This is better in a form of a browser extention
-proxy :+1:

**Pull request merged**